windows-esd-to-iso

```diff
@@ -1,30 +1,31 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-set -ex
+set -Eeuo pipefail -o posix
```

Comment:
Taking the new set options one at a time:

- "-E": not needed. We don't have any subshells here that need to use ERR traps, so no need to introduce a Bash dependency for this.
- "-u": good call.
- "-o pipefail": let's do this too. Per my previous question: I wasn't aware [this was actually POSIX now](https://pubs.opengroup.org/onlinepubs/9799919799/utilities/V3_chap02.html#tag_19_26). Thanks for giving me an impetus to look that up 🙂 
- "-o posix": not needed, because I don't want to make us dependent on Bash in the first place.

I also have been leaning on "-x" in this case as this tool's progress output. If we take it out, this tool needs to be more verbose about what it's doing.